packages/camera/camera/android/src/test/java/io/flutter/plugins/camera/features/resolution/ResolutionFeatureTest.java

```diff
@@ -72,6 +81,39 @@ public void before() {
         .thenReturn(mockProfileLow);
   }
 
+  public void beforeOn31() {
```

Comment:
Missed this before; same note here as earlier: before->beforeLegacy, beforeOn32->before